downloads/windows.markdown

```diff
@@ -0,0 +1,106 @@
+---
+title: Downloads for Windows
+page: downloads
+isDownloads: true
+---
+
+# Downloads for Windows
+
+## Package-based install
+
+The package based installers on Windows use [chocolatey](https://chocolatey.org).
+To use them first [configure Chocolatey](https://chocolatey.org/install) on your machine.
+
+### Setting up a new environment
+
+Follow the steps for setting up ghc and cabal given by [ghc chocolatey](https://chocolatey.org/packages/ghc)
+
+For new installations, it is recommended to use `Haskell-Dev` which will also setup and configure `MSys2` for use with `GHC` and `Cabal`.
+
+**If upgrading from `Haskell-Platform` please see instructions below.**
+
+`Haskell-Dev` can be installed from Chocolatey as follows from an elevated command prompt:
+
+```bash
+choco install haskell-dev
+refreshenv
+```
+
+Chocolatey has limited [non-administrative](https://chocolatey.org/docs/installation#non-administrative-install) install support for those that absolutely need it.
+
+After this you can upgrade or install additional `ghc` versions independently (see below).
+
+For integrating with Cloud CI systems, see [haskell-dev](https://hub.zhox.com/posts/introducing-haskell-dev/). 
+
+Packages are installed into `%ProgramData%\chocolatey\lib\ghc\` and `%ProgramData%\chocolatey\lib\cabal\`
+
+Steps to setup stack are [on the stack website](https://docs.haskellstack.org/en/stable/install_and_upgrade/#ubuntu).
+
+### Upgrading from Haskell-Platforms
+
+If moving from a `Haskell-Platform` based installation to `Chocolatey` you will need to clean your `cabal` configuration before uninstalling `Haskell-Platform`.
+
+To do this run:
+```bash
+cabal user-config init -f
+```
+
+and proceed to uninstall `Haskell-Platform` before installing using Chocolatey. Otherwise, the Chocolatey packages may not modify the Cabal configuration file.
+
+Do not attempt to use both `Haskell-Platform` and `Chocolatey`-based installations at the same time, as they modify the same global configuration.
+
+### Upgrading/Downgrading GHC or Cabal
+
+`GHC` or `Cabal` can be upgraded using the [Chocolatey upgrade](https://chocolatey.org/docs/commandsupgrade) command.
+
+For `GHC` (which may or may not trigger a cabal upgraded based on the version constraints)
+
+```bash
+choco upgrade ghc
+refreshenv
+```
+
+For just `cabal` which will never trigger a `ghc` upgrade.
+```bash
+choco upgrade cabal
+refreshenv
+```
+
+For both `ghc` and `cabal`
+
+```bash
+choco upgrade ghc cabal
+refreshenv
+```
+
+Downgrades can be performed by specifying the version number during installation. e.g:
+
+```bash
+choco install ghc --version 8.6.5
+refreshenv
+```
+
+A list of available versions can be found on the [GHC Chocolatey](https://chocolatey.org/packages/ghc) website or by
+using the command
+
+```bash
+choco list ghc --all
+```
+
+### Installing multiple versions of GHC
+
+Multiple versions of `GHC ` can be installed concurrently by using the `-m` switch:
```

Comment:
`-m` is short for `Multiple` it just lets the package manager allow more than one version of the same package to be installed at the same time.  Normally this would result in an error saying you have a more recent/older version install and to upgrade or downgrade instead..

Reply:
I would use one of the long aliases for that Param: `--allow-multiple` for example